docs/modules/ROOT/pages/containers.adoc

```diff
@@ -196,6 +196,7 @@ TIP: If your project is under version control, we recommend using a diff tool to
 |MLflow                                 | `mlflow-deploy-v2`                 |https://github.com/boozallen/aissemble/blob/{git-tree}/extensions/extensions-helm/aissemble-mlflow-chart/README.md[MLFlow,role=external,window=_blank]
 |Policy Decision Point                  | `policy-decision-point-deploy-v2`  |https://github.com/boozallen/aissemble/blob/{git-tree}/extensions/extensions-helm/aissemble-policy-decision-point-chart/README.md[Policy Decision Point,role=external,window=_blank]
 |S3 Local                               | `s3local-deploy-v2`                |https://github.com/boozallen/aissemble/blob/{git-tree}/extensions/extensions-helm/aissemble-localstack-chart/README.md[S3 Local,role=external,window=_blank]
+|Spark-Infrastructure                   | `aissemble-spark-infrastructure-deploy-v2 | https://github.com/boozallen/aissemble/lob/{git-tree}/extensions/extensions-spark-infrastructure/[Spark Infrastructure,role=external,window=_blank]
```

Comment:
`aissemble/lob/{git-tree}` -> `aissemble/blob/{git-tree}`

Reply:
Looks like you're missing the README on the end too

Reply:
The Readme is tricky since it's actually three Readmes. Do we need a parent readme that links to the other three?

Reply:
Fixed.